docs/workflow/debugging/mono/wasm-debugging.md

```diff
@@ -257,3 +257,87 @@ C @ blazor.webassembly.js:1
 dispatchGlobalEventToAllElements @ blazor.webassembly.js:1
 onGlobalEvent @ blazor.webassembly.js:1
 ```
+
+# Enabling additional logging in Blazor
+
+In .NET 8+, Blazor startup can be controlled by setting the `autostart="false"` attribute on the
+`<script>` tag that loads the blazor webassembly framework.  After that, a call to the
+`globalThis.Blazor.start()` JavaScript function can be passed additional configuration options,
+including setting mono environment variables, or additional command line arguments.
+
+The name of the script and the location of the `<script>` tag depends on whether the project was a
+Blazor WebAssembly project (template `blazorwasm`) or a Blazor project (template `blazor`).
+
+See the runtime `DotnetHostBuilder` interface in
+[dotnet.d.ts](../../../../src/mono/wasm/runtime/dotnet.d.ts) for additional configuration functions.
+
+## Blazor WebAssembly
+
+In a `blazorwasm` project, the script is `_framework/blazor.webassembly.js` and it is loaded in `wwwroot/index.html`:
+
+```html
+<body>
+    <div id="app">
+      ...
+  </div>
+
+  <div id="blazor-error-ui">
+    ...
+  </div>
+    <script src="_framework/blazor.webassembly.js"></script>
+</body>
+```
+
+Replace it with this:
+
+```html
+<body>
+    <div id="app">
+        ...
+    </div>
+
+    <div id="blazor-error-ui">
+        ...
+    </div>
+    <script src="_framework/blazor.webassembly.js" autostart="false"></script>
+
+    <script>
+        Blazor.start({
+            configureRuntime: dotnet => {
+                dotnet.withEnvironmentVariable("MONO_LOG_LEVEL", "debug");
+                dotnet.withEnvironmentVariable("MONO_LOG_MASK", "all");
+            }
+        });
+    </script></body>
+```
+
+## Blazor
```

Comment:
Is it worth having two snippets if they diff only in script name? What about just mentioning that?

Reply:
That's not the only difference. the argument to `Blazor.start` is different.  the `blazor` project needs an extra `webAssembly:` dictionary

Reply:
Thanks! I missed that